client/js/config/models.js

```diff
@@ -0,0 +1,44 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+export const toggleOnbording = async () => {
+  try {
+    return await AsyncStorage.setItem(`onBoarding`, true);
```

Comment:
you should be toggling the value (false -> true/true -> false) here since the name of the function. If you want to keep the value to true and not toggle, change the name of the function. 

Use a descriptive key name (ex, isOnBoarded, onBoardingCompleted, etc).

Reply:
## Fixed